doc/rippled-example.cfg

```diff
@@ -423,11 +423,52 @@
 #
 #   retry_sequence_percent = <number>
 #
-#       If a client resubmits a transaction, the new transaction's fee
+#       If a client resubmits a transaction (same sequence number as a
+#       transaction already in the queue), the new transaction's fee
 #       must be more than <number> percent higher than the original
 #       transaction's fee, or meet the current open ledger fee to be
-#       considered. Default: 125.
+#       considered. Default: 25.
 #
+#   multi_txn_percent = <number>
+#
+#       If a client submits multiple transactions (different sequence
+#       numbers), later transactions must pay a fee at least <number>
+#       percent higher than the transaction with the previous sequence
+#       number.
+#       Default: -90.
```

Comment:
It's the default value I'm using (https://github.com/ximinez/rippled/blob/fee-multi/src/ripple/app/misc/TxQ.h#L58-L59), and frankly, I'm debating getting rid of it entirely.